test/passport_test.coffee

```diff
@@ -53,6 +66,89 @@ describe "Register / Login", ->
                 body.isAuthenticated.should.be.ok
                 done()
 
+    describe "Login with two-factors authentication: TOTP", ->
+        
+        before (done) ->
+            # Logging out
+            client.get "logout", (error, response, body) ->
+                # Adding TOTP setting to the user's settings
+                user.request "all", (error, users) ->
+                    user.updateAttributes users[0]._id,
```

Comment:
What says the documentation is that you have to call it on the class object, not on an instance. In case of instance, the id argument should be removed.